packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -720,7 +730,7 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
                         pushAppView={pushAppView}
                         currentAppView={currentAppView}
                         appViews={appStates}
-                        onRouteFinished={() => resetStateAndUI()}
+                        onRouteFinished={() => onRouteFinish()}
```

Comment:
Since we no longer use the resetStateAndUI to clean up then the route is finished, there is maybe no need to call onRouteFinish from the MapTemplate component. There is maybe an opportunity to eliminate some callback spaghetti by just using the hook from the Directions component